base/src/main/res/layout/item_feed_card.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
ılk gunden xml dosyalarında bütün margin padding width height degerleri guide' a göre ilerlesin, hardcoded olmasın daha sonra buraları refactor etmeyiz :)

Reply:
Aslında şöyle ki adım adım attık dün. Bozulma olabilmesi durumunda geri çekebilmek için.
Şu an textlerin style ı eksik 3 adet daha eklenecek. 
Card backgroundu için ekleme yapılacak.
Tüm renkler colors'a eklenmedi.
Ama sadece o xml'deki, spesifik bir item ait olan bir değerin style a çekilmesine gerek yok sanki.

Reply:
Bu arada ben yaptıkça temizlendikçe master'a merge edilebilir düşüncesinde dün pr'ı açtım ama @Murat ve @Erol eğer böyle içinize sinmiyorsa, açık kalsın az önce yazdığım maddeleri yapınca, son haline bakın.